ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeRootNodeExecutor.java

```diff
@@ -291,7 +291,7 @@ private boolean shouldExecuteNextLeaf()
       }
       else
       {
-         return !currentlyExecutingLeaves.isEmpty();
+         return currentlyExecutingLeaves.isEmpty();
```

Comment:
I broke this on Dec 10th
![image](https://github.com/user-attachments/assets/43ca42a7-86a4-4ba8-b6a5-8eb0ad51ba9d)

Reply:
The logic is supposed to be "if no actions" are executing.

Reply:
currentlyExecutingLeaves being empty is already equivalent to !anyActionExecuting, so the '!' was an error